components/AutoComplete/AutoComplete.jsx

```diff
@@ -0,0 +1,356 @@
+      wrapperRef.current.children[1].focus();
+      setCursor(0);
+    }
+  };
+
+  const generateSuggestions = () =>
+    showSuggestions && filterSuggestions ? (
+      <List
+        theme={theme}
+        role="listbox"
+        id="autocompleteOptions"
+        ref={listOptions}
+      >
+        {filterSuggestions.map((item, index) => (
+          <ListItens
+            key={item}
+            aria-posinset={index}
+            onClick={() => handleItemClick(item)}
+            theme={theme}
+            aria-selected={index === cursor}
+            role="option"
+            tabIndex="-1"
+          >
+            {item}
+          </ListItens>
+        ))}
+      </List>
+    ) : null;
+
+  const generateAssistiveDescript = () => {
+    if (showSuggestions) {
+      return `${filterSuggestions.length} estão disponiveis. ${
+        filterSuggestions[cursor]
+      } ${cursor + 1} de ${filterSuggestions.length} está destacado`;
+    }
+    return `${filterSuggestions.length} estão disponiveis.`;
+  };
+
+  const downPress = useKeyPress('ArrowDown');
+  const upPress = useKeyPress('ArrowUp');
+  const enterPress = useKeyPress('Enter');
+  const tabPress = useKeyPress('Tab');
+
+  /* istanbul ignore next */
+  useEffect(() => {
+    if (showSuggestions && filterSuggestions.length && downPress) {
+      const selectedCursor =
+        cursor < filterSuggestions.length - 1 ? cursor + 1 : cursor;
+      setCursor(selectedCursor);
+      listOptions.current.children[selectedCursor].focus();
+    }
+  }, [downPress]);
+
+  /* istanbul ignore next */
+  useEffect(() => {
+    if (showSuggestions && filterSuggestions.length && upPress) {
+      const selectedCursor = cursor > 0 ? cursor - 1 : cursor;
+      setCursor(selectedCursor);
+      listOptions.current.children[selectedCursor].focus();
+    }
+    if (upPress && cursor === 0) {
+      wrapperRef.current.children[1].focus();
+      setCursor(-1);
+    }
+  }, [upPress]);
+
+  /* istanbul ignore next */
+  useEffect(() => {
+    if (showSuggestions && filterSuggestions.length && enterPress) {
+      setUserTypedValue(filterSuggestions[cursor]);
+      setShowSuggestions(false);
+      wrapperRef.current.children[1].focus();
+    }
+  }, [cursor, enterPress]);
+
+  /* istanbul ignore next */
+  useEffect(() => {
+    if (showSuggestions && tabPress) {
+      setUserTypedValue(filterSuggestions[cursor]);
+      setShowSuggestions(false);
+    } else if (tabPress) {
+      setShowSuggestions(false);
+    }
+  }, [tabPress]);
+
+  useEffect(() => {
+    window.addEventListener('click', handleClickOutside);
+    window.addEventListener('keydown', handleEscPress);
+    return () => {
+      window.removeEventListener('click', handleClickOutside);
+      window.removeEventListener('keydown', handleEscPress);
+    };
+  }, []);
+
+  return (
+    <Wrapper ref={wrapperRef}>
+      <InputLabel htmlFor={id} error={error}>
+        {label}
+      </InputLabel>
+      <InputText
```

Comment:
clean icon input text is not align:

![2022-01-04_17-06](https://user-images.githubusercontent.com/190265/148117367-1c149a59-28d3-4022-8948-2b8e3831c601.png)